user.php

```diff
@@ -0,0 +1,61 @@
+<?php
+	include("connection.php");
+	/**
+	* @author: mfsi_krishnadev
+	* @class: validating user
+	*/
+	class user extends db_connection
+	{
+		
+		/*
+		 * @access: public
```

Comment:
tab and space, both are used in multiple files.
either use tab or space but not both.
better to use spaces and 4 space.